controllers/namespacescope/namespacescope_controler_test.go

```diff
@@ -102,6 +114,14 @@ var _ = Describe("NamespaceScope controller", func() {
 				}
 				return len(nss.Spec.NamespaceMembers)
 			}, timeout, interval).Should(Equal(3))
```

Comment:
@horis233 

May I ask why we expect `len(NamespaceMembers)==3`?

From my understanding, the controller will add the `namespace` of all `OperandRequest` and `operator.namespace` in `OperandRegistry`.

So in this case, there is one `OperandRequest` and `namespace` of two operators `etcd` and `jenkins` has the same. The total number of namespace should be `2`.

Reply:
It's because it added the namespace of the operator as well.
You can use `fmt` to print the namespace list to check.

Reply:
Check it, indeed there are three namespaces